documentation/asciidoc/accessories/iqaudio/dac_plus.adoc

```diff
@@ -1,15 +1,15 @@
 === Raspberry Pi DAC{plus}
 
-Raspberry Pi DAC{plus} is a high-resolution audio output HAT that provides 24 bit 192 kHz digital audio output.
+Raspberry Pi DAC{plus} is a high-resolution audio output HAT that provides 24-bit 192kHz digital audio output.
 
 image::images/DAC+_Board_Diagram.jpg[width="80%"]
 
-A Texas Instruments PCM5122 is used in the DAC{plus} in order to deliver analogue audio to the Phono connectors of the device. It also supports a dedicated headphone amplifier and is powered by the Raspberry Pi through the GPIO header.
+A Texas Instruments PCM5122 is used in the DAC{plus} to deliver analogue audio to the phono connectors of the device. It also supports a dedicated headphone amplifier and is powered via the Raspberry Pi through the GPIO header.
 
 ==== Pinouts
 [cols="1,12"]
 |===
-| *P1* | Analogue out (0..2V RMS), carries GPIO27, MUTE signal (headphone detect), Left and Right
-Audio and Left and Right Ground.
+| *P1* | Analogue out (0.2V RMS), carries GPIO27, MUTE signal (headphone detect), left and right
```

Comment:
Another instance where I suspect this originally meant "in the range zero to two Volts".

Reply:
Fixed